components/HighlightedLinks/src/index.scss

```diff
@@ -0,0 +1,41 @@
+.denhaag-highlighted-links {
+  background-color: var(--denhaag-highlighted-links-background-color);
+  padding-block-start: var(--denhaag-highlighted-links-padding-block-start);
+  padding-block-end: var(--denhaag-highlighted-links-padding-block-end);
+  width: var(--denhaag-highlighted-links-width);
+}
+
+.denhaag-highlighted-links__container {
+  --denhaag-highlighted-links-width: var(--denhaag-highlighted-links-container-width);
+
+  margin-inline-start: var(--denhaag-highlighted-links-container-margin-inline-start);
+  margin-inline-end: var(--denhaag-highlighted-links-container-margin-inline-end);
+  max-width: var(--denhaag-highlighted-links-container-max-width);
+}
+
+.denhaag-hero--theme-page + .denhaag-highlighted-links {
+  --denhaag-highlighted-links-padding-block-start: var(
+    --denhaag-highlighted-links-hero-theme-page-highlighted-links-padding-block-start
+  );
+}
+
+.denhaag-highlighted-links__navigation-list {
+  display: var(--denhaag-highlighted-links-navigation-list-display);
+  gap: var(--denhaag-highlighted-links-navigation-list-gap);
+
+  @media (min-width: 48rem) {
+    grid-template-columns: var(--denhaag-highlighted-links-navigation-list-grid-template-columns);
+  }
+
+  @media (min-width: 68.875rem) {
```

Comment:
Overwrites of a CSS var u want to do on the top level of the element, that way the CSS output is reduced and other developers can overwrite these values more easily. #truepowerofcssvariables.

Reply:
@paulAcato This does not work in storybook:
```
  @media (min-width: var(--denhaag-highlighted-links-navigation-list-min-width)) {
    grid-template-columns: var(--denhaag-highlighted-links-navigation-list-grid-template-columns);
  }

  @media (--denhaag-highlighted-links-navigation-list-min-width: --denhaag-highlighted-links-navigation-list-min-width-2) {
    --denhaag-highlighted-links-navigation-list-grid-template-columns: var(
      --denhaag-highlighted-links-navigation-list-grid-template-columns-2
    );
  }
```

Reply:
This isn't supported on the web. Also thats not what i ment with my comment above.

What you want is the media-queries on the top level of the element ( `.denhaag-highlighted-links` ) instead of `.denhaag-highlighted-links__navigation-list`. The reason why is because if another develop want to use this component they need to overwrite it with `.denhaag-highlighted-links .denhaag-highlighted-links__navigation-list` before the css variable will react.